packages/revocation/src/implementations/revocationOnChain.ts

```diff
@@ -0,0 +1,127 @@
+import {
+  Contract, ethers, Event, utils,
+} from 'ethers';
+import { EwSigner, addressOf } from '@ew-did-registry/did-ethr-resolver';
+import { abi as RevocationOnChainAbi } from '../constants/RevocationRegistryOnChain.json';
+
+export class RevocationOnChain {
```

Comment:
Are there tests for this class? I don't see any at the moment

Reply:
Those tests are part of the IAM contract library. 

https://github.com/energywebfoundation/iam-contracts/blob/feature/EDR-25-Revocation_registry/test/RoleDefinitionResolver.testSuite.ts